torch/fx/experimental/proxy_tensor.py

```diff
@@ -817,6 +817,34 @@ def _maybe_record_pointwise_barrier(
     last_node.meta["low_precision_pointwise_barrier"] = True
 
 
+in_export = False
+
+
+@contextmanager
+def _force_copy_on_aten_to_context(
```

Comment:
This should be under a flag right?

Reply:
Also don't we need to ban the mutation on output? (same thing as last time)

Reply:
> Also don't we need to ban the mutation on output? (same thing as last time)

@tugsbayasgalan  I don't think we need to do it here. Here we're just trying to preserve the graph. The aten.to.dtype op actually has the same behavior as the eager aten.to: it'll alias or copy depending on the input.

Previously we had to do it in functionalization, because we changed the op to aten._to_copy, which always does a copy and never alias.

Reply:
> Hmm what i mean is we are intentionally diverging from eager behaviour here because we are force copying right? This will break functuonalizaiton analysis because now it will think input and output never alias each other.

This is only meant for producing the right (and device-agnostic) pre-dispatch graph. It should not change anything about the functionalization.